### PR TITLE
[Snappy] Add to oss-fuzz

### PR DESCRIPTION
**Pull request by bshastry**
- Depends on https://github.com/google/snappy/pull/82
- Will change Docker repo link once that is merged.
- Please check if primary contact email is correct

CC @pwnall @inferno-chromium @Dor1s

### CODE REVIEW

File: projects/snappy/project.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+homepage: "https://github.com/google/snappy"
+primary_contact: "victor@costan.us"
```
Comment (Dor1s) [inline projects/snappy/project.yaml]: @pwnall please check the contact here

Comment (pwnall) [reply projects/snappy/project.yaml]: `storage-dev@chromium.org` would be better

**Comment by bshastry** [comment]
@Dor1s yeah, this depends on google/snappy#82

**Comment by Dor1s** [comment]
> @Dor1s yeah, this depends on google/snappy#82

Ack, please let us know when it's ready to be merged.

**Comment by bshastry** [comment]
I think this PR can be closed now since snappy is integrated?

**Comment by inferno-chromium** [comment]
> I think this PR can be closed now since snappy is integrated?

yes!

**Pull request closed without merging**